R/redist_smc.R

```diff
@@ -155,6 +155,14 @@ redist_smc = function(map, nsims, counties=NULL, compactness=1, constraints=list
     if (!inherits(constraints, "redist_constr")) {
         constraints = new_redist_constr(eval_tidy(enquo(constraints), map))
     }
+    if (any(c('edges_removed', 'log_st') %in% names(constraints))) {
+        cli_warn('{.var edges_removed} or {.var log_st} constraint found in {.arg constraints}
+                 and will be ignored. Adjust using {.arg compactness} instead.')
```

Comment:
while we're at it, let's format this using the rlang bullets -- so 
```r
cli_warn(c("{.var edges_removed} or {.var log_st} constraint found in
           {.arg constraints} and will be ignored.",
           ">"="Adjust using {.arg compactness} instead."))
```

Reply:
Sure. Updated